tests/llmcompressor/transformers/compression/recipes/sparse_int8.yaml

```diff
@@ -0,0 +1,46 @@
+pruning_stage:
+    obcq_modifiers:
+        SparseGPTModifier:
+            sparsity: 0.5
+            sequential_update: true
```

Comment:
I would also add a 2:4 sparse only recipe for testing

Reply:
Added!